man/org.freedesktop.sysupdate1.xml

```diff
@@ -489,9 +489,9 @@ node /org/freedesktop/sysupdate1/job/_1 {
 
       <para>The <varname>Id</varname> property exposes the numeric job ID of the job object.</para>
```

Comment:
i'd probably still do this out of process. i think we might eventually want to hook more stuff into this, for example, automatically enable features based on the udev db, but if that's the case things that are "fast" right now, might eventually not be quite as fast anymore.

hence, doesn't matter, but given you already are doing this out-of-prcess, stick to it.